lists/ua.csv

```diff
@@ -954,3 +953,51 @@ https://ok.ru/,GRP,Social Networking,2017-12-16,oonitarian,sanction list
+http://tnt-online.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://tvzvezda.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://ren.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://russia.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://5-tv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://rt.com/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://life-news.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://1tv.com/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://tvc.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://rtr-planeta.com/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://ntvmir.ntv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://tv.rbc.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://istoriya.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://365days.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://techno24.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://mir24.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://strana-tv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://russkiyillusion.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://domkino.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://oruzhie.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://illusionplus.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://fenixplus.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://khl.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://telecafe.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://sarafan-tv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://naukatv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://kuhnyatv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://autoplustv.com/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://moya-planeta.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://shanson.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://cinetv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://sovsekretno.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://strashnoe.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://muz1.tv/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://hdlife.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://tdktv.com/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://24doc.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://telekanaldetskiy.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://stvchannel.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://indiatv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://laminortv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://rtgtv.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://hdkinoklub.com/,NEWS,News Media,2017-01-27,oonitarian,SBU list
+http://tv-stream.ru/,NEWS,News Media,2017-01-27,oonitarian,SBU list
```

Comment:
Are you sure this website (and others submitted here), should not fall under the "Media sharing" category? Some of the websites in this list don't seem to be media outlets, but rather video sharing websites.

Reply:
It's true that hdkinoclub.com and tv-stream are Media sharing websites. However, rtgtv and laminortv are TV channels (laminor is a channel about music, rtgtv is a group of cable TV channels). Which category would be most appropriate for this?

Reply:
Yeah you can leave those ones as news, I am suggesting you change the ones that are media sharing to media sharing.

Reply:
I just made a commit editing the category codes based on this discussion.